app/src/test/java/com/gfabrego/moviesapp/popular/list/PopularShowsPresenterTest.kt

```diff
@@ -36,10 +41,13 @@ class PopularShowsPresenterTest {
         val shows = anyListOfShows()
         val request = PageRequest.Paged(1)
         given(pageRequestFactory.createInitialPage()).willReturn(request)
-        given(getPopularShows.build(GetPopularShows.Params(request))).willReturn(flowOf(PopularShowsResponse(shows, PageRequest.Paged(2), -1)))
+        given(getPopularShows.build(GetPopularShows.Params(request))).willReturn(
+            flowOf(PopularShowsResponse(shows, PageRequest.Paged(2), -1))
+        )
 
-        buildPresenter().attachView()
+        buildPresenter().attachView().join()
```

Comment:
IMHO is better to use `testScope.advanceTimeBy(5000)` instead of `join()`, this way we can also ensure that the expected time has passed.